src/ucp/wireup/wireup.c

```diff
@@ -1803,5 +1803,44 @@ unsigned ucp_ep_init_flags(const ucp_worker_h worker,
     return flags;
 }
 
+double ucp_wireup_iface_lat_distance_v1(ucp_context_h context,
```

Comment:
maybe pass worker instead of context to reduce some code when this function is called?